app/Platform/Commands/SyncGameSets.php

```diff
@@ -28,11 +28,16 @@ public function handle(): void
     {
         $this->info('Running a full GameAlternatives -> game_sets sync...');
 
+        Cache::forget('central_hub_id');
+
         // This will be a full reset. Delete any existing game_sets data.
+        // We'll use TRUNCATE to reset the auto-incrementing ID counter back to 1.
         $this->info("\nDeleting any existing game_sets data...");
-        GameSetLink::query()->forceDelete();
-        GameSetGame::query()->forceDelete();
-        GameSet::query()->forceDelete();
+        DB::statement('SET FOREIGN_KEY_CHECKS=0;');
+        DB::statement('TRUNCATE TABLE game_set_links;');
+        DB::statement('TRUNCATE TABLE game_set_games;');
+        DB::statement('TRUNCATE TABLE game_sets;');
```

Comment:
You can use `GameSet::truncate()` for this.

Reply:
Done in latest.